cmd/virt-launcher/sh.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/sh.orig
+
+if [ ${TERM} = "xterm" ] ; then
+  namespace="$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace)"
+  name="$(ls /var/run/kubevirt-private/${namespace}/)"
+  /usr/bin/sh.orig -c "/sock-connector /var/run/kubevirt-private/${namespace}/${name}/virt-serial0"
```

Comment:
Does this command need to account for "$@"?

Reply:
No, we can't forward arguments here.